src/features/projects/components/AddToBallot.tsx

```diff
@@ -47,7 +49,7 @@ export const ProjectAddToBallot = ({ id, name }: Props) => {
         </IconButton>
       ) : (
         <Button
-          disabled={!address}
+          disabled={!address || !isRegistered}
```

Comment:
If user is not allowed to vote, this button should be hidden, not disabled.

Reply:
Take a look at my PR #21. It's implemented there.

Reply:
I found it, so what should I do now?